src/pages/home/report/ReportActionCompose.js

```diff
@@ -591,6 +592,10 @@ class ReportActionCompose extends React.Component {
         const commentAfterColonWithEmojiNameRemoved = this.state.value.slice(this.state.selection.end).replace(CONST.REGEX.EMOJI_REPLACER, CONST.SPACE);
 
         this.updateComment(`${commentBeforeColon}${emojiCode} ${commentAfterColonWithEmojiNameRemoved}`, true);
+        // Clear the contents of a TextInput in some android keyboards
+        if (RNTextInputReset) {
+            RNTextInputReset.resetKeyboardInput(findNodeHandle(this.textInput));
```

Comment:
I think it's better if we just straight use `NativeModules.RNTextInputReset` so other dev will get the idea of where `RNTextInputReset` comes from.